pkgs/applications/networking/testssl/default.nix

```diff
@@ -0,0 +1,51 @@
+{ stdenv, fetchFromGitHub, pkgs, lib }:
+
+stdenv.mkDerivation rec {
+  version = "2.9.5-1";
+  name = "testssl.sh-${version}";
+
+  src = fetchFromGitHub {
+    owner = "drwetter";
+    repo = "testssl.sh";
+    rev = "v${version}";
+    sha256 = "0hz6g685jwl0c0jrdca746425xpwiwc8lnlc2gigga5hkcq8qzl9";
+  };
+
+  buildInputs = with pkgs; [
```

Comment:
Nit: `makeWrapper` is build-time only and could be listed in `nativeBuildInputs` instead.